sway-utils/src/performance.rs

```diff
@@ -29,9 +29,9 @@ macro_rules! time_expr {
                 if cfg.metrics_outfile.is_some() {
                     #[cfg(not(target_os = "macos"))]
                     let memory_usage = {
-                        use sysinfo::{System, SystemExt};
+                        use sysinfo::{MemoryRefreshKind, System};
                         let mut sys = System::new();
-                        sys.refresh_system();
+                        sys.refresh_memory_specifics(MemoryRefreshKind::new().with_ram());
```

Comment:
Why is this changing? It looks like [refresh_memory](https://docs.rs/sysinfo/latest/sysinfo/struct.System.html#method.refresh_memory) is still available in the latest version.

Reply:
The `refresh_system()` method was removed in a newer `sysinfo` version and so I looked for a suitable replacement. Although `refresh_memory()` exists, the line following the refresh 
call is `sys.used_memory()`, returning the total `ram` used. Seeing as total `ram` doesn't include `swap` values, I figured that we could possibly save cycles by skipping swap calculations that we never end up reading.

Looking at `sysinfo` internals, Linux provides a nice interface via `/proc/meminfo` which is read once as a whole, but all other platforms end up doing separate system calls to retrieve swap info if swap is set to refresh.